bibtex-actions.el

```diff
@@ -172,6 +172,7 @@ and nil means no action."
 (defcustom bibtex-actions-major-mode-functions
   '(((latex-mode) .
      ((local-bib-files . bibtex-actions-latex--local-bib-files)
+      (insert-keys . bibtex-actions-latex--insert-keys)
```

Comment:
What should `insert-keys` be for `org`? Bibtex-completions uses the comma separated list for everything, but does the org-cite reqiuires `@` in front of the key?

Reply:
Both pandoc and org-cite use the `@key` syntax, with semi-colon delimiter. 

See [bibtex-completion-format-citation-pandoc-citeproc](https://github.com/tmalsburg/helm-bibtex/blob/b85662081de98077f13f1a9fac03764702325d28/bibtex-completion.el#L1024).

But that has had me wonder whether insert-keys is a useful command; particularly in org-cite (where `org-cite-insert` works differently depending on where point it).

Is it useful in latex, given insert-citation?

Reply:
> Both pandoc and org-cite use the `@key` syntax, with semi-colon delimiter.

I added that. 

> 
> But that has had me wonder whether insert-keys is a useful command; particularly in org-cite (where `org-cite-insert` works differently depending on where point it).
> 
> Is it useful in latex, given insert-citation?

It is useful for me because `insert-citation` gives a menu with cite commands and I only use `\cite` so I have defined a command that use `\cite` using `insert-keys`.

I want to change `insert-citation` so that it can be passed the citation argument as an optional argument so that it gives you the menu only if it is not passed, in that case it will become less useful.

Reply:
> It is useful for me because `insert-citation` gives a menu with cite commands and I only use `\cite` so I have defined a command that use `\cite` using `insert-keys`.
> 
> I want to change `insert-citation` so that it can be passed the citation argument as an optional argument so that it gives you the menu only if it is not passed, in that case it will become less useful.

Yeah, this is what I meant about `org-cite-insert`.

I'm not sure it's the 100% best UI ATM, but it's an improvement. So:

1. when you run it in bare text, it just inserts the citation with keys.
2. when you run it when point is on "cite", it gives you the style UI
3. when you run it when point is on a citekey, it prompts to replace the key

Not sure how much of that is possible in LaTeX, given the more lightweight syntax, but might be worth thinking about.